caffe2/operators/utility_ops.h

```diff
@@ -266,33 +266,60 @@ class SumOp : public Operator<Context> {
           true /*async*/);
       return true;
     }
+    // initialize output
     auto* output = Output(0, input0.sizes(), at::dtype<T>());
     T* output_data = output->template mutable_data<T>();
+    std::vector<int> output_dims = GetDimsVec(*output);
```

Comment:
Don't we need to compute the new output size based on the broadcasting? And resize the output?